src/common/services/session/sessionModal.service.js

```diff
@@ -46,12 +46,34 @@ const SessionModalService = /* @ngInject */ function ($uibModal, $log, modalStat
       currentModal.result
         .then(angular.noop, () => {
           analyticsFactory.track(options.dismissAnalyticsEvent)
+          switch (type) {
+            case 'sign-in':
+            case 'sign-up':
+              analyticsFactory.trackGTM('ga-sign-in-exit')
+              break
+            case 'user-match':
+              analyticsFactory.trackGTM('ga-registration-exit')
+              break
+            default:
+              break
+          }
         })
     }
 
     if (options.openAnalyticsEvent) {
       currentModal.opened.then(() => {
         analyticsFactory.track(options.openAnalyticsEvent)
+        switch (type) {
+          case 'sign-in':
+          case 'sign-up':
```

Comment:
I didn't know if these needed to be two separate events or not. There wasn't anything in the documentation so I just used the same analytics call for both. I'm going to bring it up to Clark though.